STEER/Utilities/alihadd.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
```

Comment:
Use `#!/bin/bash` everywhere. We want it to be compatible with the system's one, not a generic bash.